charts/spire/charts/spire-agent/README.md

```diff
@@ -12,13 +12,17 @@ A Helm chart to install the SPIRE agent.
 |-----|------|---------|-------------|
 | bundleConfigMap | string | `"spire-bundle"` |  |
 | clusterName | string | `"example-cluster"` |  |
+| extraContainers | list | `[]` |  |
+| extraVolumeMounts | list | `[]` |  |
+| extraVolumes | list | `[]` |  |
```

Comment:
The use of `extra` here feels like these items, if set, would be superfluous and not needed but I imagine if someone is setting these they're doing so because the volumes and mounts are very much needed. Is there a better term we can use other than `extra`? 

`additional` comes to mind.